src/components/Sidebar/Sidebar.css

```diff
@@ -1,3 +1,9 @@
-.tlbx-sidebar {
-  padding: 2em;
+.toolbox-sidebar {
```

Comment:
I decided to rename Toolbox classes with a `tlbx` prefix, to avoid any conflicts with our real life projects. To be decided if we keep it that way.